src/pint/data/runtime/observatories.json

```diff
@@ -1399,7 +1515,8 @@
             1665449.2317,
             -2505096.8054
         ],
-        "origin": "H.E.S.S., the High Energy Stereoscopic System, an Imaging Atmospheric Cherenkov Telescope.\nThese coordinates are provided (from geodetic coordinates : lat = 23°16'18''S, lon = 16°30'00''E at 1800m asl) by the Maxime Regeard and Arache Djannati-Atai on behalf of the H.E.S.S. Collabotation, 2023 January 18."
+        "fullname": "H.E.S.S., the High Energy Stereoscopic System",
```

Comment:
The High Energy Stereoscopic System (HESS)

Reply:
fixed